.gitignore

```diff
@@ -84,3 +84,4 @@ build/
 
 # VS Code files
 .vscode
+docs/guides/stay-up-to-date/index.md
```

Comment:
We actually don't want to do this as that will make it much harder to change anything on this page if we ever want to do that. 

Instead we need to either architect the `docs-sourcer` to allow this kind of functionality (ie - mixing auto-generated and manually written content on a page) or we need to update our IA such that this content gets separated.

Reply:
Ah, good point. Strangely, this `.gitignore` didn't work anyway. I think having a fully auto-generated page is the way to go.

Separately, it looks like https://docs.gruntwork.io/guides/stay-up-to-date/#gruntwork-releases is now hidden from the IA. Perhaps we should add a new page under Library called "Release Notes" that includes the releases section in https://docs.gruntwork.io/guides/stay-up-to-date/#gruntwork-releases?